Add presto file (#178)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/linbit.md

---
name: Linbit
category: Storage
description: LINBIT is a company that provides open-source storage software DRBD and LINSTOR.
download_url: https://linbit.com/linbit-software-download-page-for-linstor-and-drbd-linux-driver/
works_on_arm: true
supported_minimum_version:
    version_number: 9.0
    release_date: 14/06/2021

optional_info:
    homepage_url: https://linbit.com/
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content:
        partner_content: https://amperecomputing.com/briefs/LINBITSDS-Brief
        official_docs: https://linbit.com/drbd-user-guide/linstor-guide-1_0-en/
    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum:
    release_notes__recommended_minimum:

>     other_info: There are no release notes present that mentions arm64 support although the official documentation mentions it, please refer: https://linbit.com/drbd-user-guide/drbd-guide-9_0-en/#_cpu_requirements.

Author: ```suggestion
    other_info: There are no release notes available for Linux/ARM64 support, although the [official documentation](https://linbit.com/drbd-user-guide/drbd-guide-9_0-en/#_cpu_requirements) mentions it.
```
